mautic/mautic
# This fixes an issue where the places map did not render for contacts

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | Y
| New feature?  | N 
| BC breaks?    |  N
| Deprecations? | N
| Fixed issues  |  

## Description

When clicking the places tab for a contact, the map remained small and JS errors resulted.

## Steps to reproduce the bug (if applicable)

Go to a contact that has an IP address associated. Click the places tab.

## Steps to test this PR

Apply the PR, repeat the above. The map should render full size and with markers for the places coordinates. View the dashboard's contact map and the countries should be completely highlighted.
